src/psyclone/tests/alggen_test.py

```diff
@@ -382,7 +408,6 @@ class TestAlgGenClassDynamo0p1(object):
     generate function as parse and PSyFactory need to be called before
     AlgGen so it is simpler to use this'''
 
-    @pytest.mark.xfail(reason="unknown")
     def test_single_invoke_dynamo0p1(self):
```

Comment:
I noticed this test is for algorithm generation for Dynamo 0.1 API (the entire `class TestAlgGenClassDynamo0p1(object)` is). 
There is a `pylint` warning (see above) and modified tests normally need to be updated to enable compilation. However, I am not sure how much this makes sense here as there is an issue to remove Dynamo 0.1 API (#282). Is there intention to re-purpose this tests for Dynamo 0.3 API (as well as the test `test_zero_invoke_dynamo0p1` below)? If so, it would make sense to a) Make them independent of the class they live in (and remove the class) for consistency with other tests, b) Update docstrings and 3) Enable both for compilation.

Reply:
We only support compilation testing for the Dynamo0.3 API so that's not an option here. Strangely pylint didn't complain for me but I've now done away with the `TestAlgGenClassDynamo0p1` class altogether. I haven't got as far as thinking about what happens when we remove Dynamo 0.1 - this is just about getting rid of the xfailing tests :-)